From bbdaa724ef363e34bd19fb2308739e59a9e8ddc8 Mon Sep 17 00:00:00 2001
From: jtsiomb <jtsiomb@7f0cb862-5218-0410-a997-914c9d46530a>
Date: Tue, 26 Mar 2019 18:55:42 +0000
Subject: [PATCH] Applied Christian's change to drop unnecessary dependency to
 X11 libraries we don't actually use, by asking cmake to link with X11_X11_LIB
 instead of the catch-all X11_LIBRARIES.

git-svn-id: http://svn.code.sf.net/p/freeglut/code/trunk/freeglut/freeglut@1845 7f0cb862-5218-0410-a997-914c9d46530a
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d1c20e8..e324a02 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -312,8 +312,8 @@ ENDIF(CMAKE_COMPILER_IS_GNUCC)
 INCLUDE(CheckIncludeFiles)
 IF(UNIX AND NOT(ANDROID OR BLACKBERRY OR FREEGLUT_WAYLAND))
     FIND_PACKAGE(X11 REQUIRED)
-    INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
-    LIST(APPEND LIBS ${X11_LIBRARIES})
+    INCLUDE_DIRECTORIES(${X11_X11_INCLUDE_PATH})
+    LIST(APPEND LIBS ${X11_X11_LIB})
     IF(X11_Xrandr_FOUND)
         SET(HAVE_X11_EXTENSIONS_XRANDR_H TRUE)
         LIST(APPEND LIBS ${X11_Xrandr_LIB})
-- 
2.19.2

